src/aics-image-viewer/components/StyleProvider/index.tsx

```diff
@@ -256,6 +262,12 @@ const CssProvider = styled.div<{ $theme: AppTheme }>`
     color: var(--color-button-icon-disabled-text);
   }
 
+  .btn-active {
+    color: var(--color-button-icon-activated-text);
+    background-color: var(--color-button-icon-activated-bg);
+    border: 1px solid var(--color-button-icon-activated-outline);
+  }
+
```

Comment:
As far as I recall, this is a specific class name given to buttons in the toolbar. Any advantages or concerns with including it in a global style like this?

(This is not to imply that you ought to change it, in general I've been frustrated by how little this app has built a unified design system into the code, just curious about this specific case)

Reply:
In this case it does make more sense to tuck this into the toolbar. If we put it here in global styling, it's possible that if the classname changes or the style breaks in the future no one will think to remove this.

The ideal case for me actually is to have IconButton or ToggleButton be its own component in its own module and govern all of its internal styling. I don't want to make that change right now, but I'm doing something similar for the Header in a later PR because the Primary/Secondary/Tertiary button styles in our figma design doc doesn't match up 1:1 with how Ant defines buttons.